CONTRIBUTING.adoc

```diff
@@ -76,7 +76,7 @@ added after the original pull request but before a merge.
 
 == Working with the Code
 If you don't have an IDE preference we would recommend that you use
-https://spring.io/tools/sts[Spring Tools Suite] or
+https://spring.io/tools[Spring Tool Suite] or
```

Comment:
Maybe this is better?
```suggestion
https://spring.io/tools[Spring Tools] or
```

Reply:
I think that `Spring Tool Suite` is the better name here. It's the name of the Eclipse distribution that includes Spring Tools 4 and that's what we're referring to. I can take care of changing it back as part of the merge.